viziphant/gpfa.py

```diff
@@ -119,7 +119,7 @@ def plot_dimensions_vs_time(returned_data,
                             gpfa_instance,
                             dimensions='all',
                             orthonormalized_dimensions=True,
-                            n_trials_to_plot=20,
+                            trials_to_plot=np.arange(20),
```

Comment:
I understand `np.arange(20)` was chosen since this is the old value and using the default will not break old code (and that's fine with me).

Do you think using `'all'` as a default would be a good idea?

Since the parameter was renamed, breaking changes are introduced anyways (except if the parameter is not supplied and the default is used).